### ISSUE TITLE
Accessibility: Clear button in radio group doesn't have aria-label

### ISSUE THREAD

**Issue by andrewtelnov** [surveyjs/survey-creator#7358]
aria-label attribute is missed for "Clear" button in radio group question.
The issue reported is [here](https://surveyjs.answerdesk.io/internal/ticket/details/T26024),
